swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+
+    def __init__(self, id = None, fabric = "Unknown", condition = 0.0, age = 0.0):
+        super().__init__(id, condition, age)
+        self.fabric = fabric
+    
+    def __str__(self):
+        return (f"An object of type {self.get_category()} with id {self.id}. " 
+                f"It is made from {self.fabric} fabric.")
```

Comment:
The first sentence in the string you return looks __very__ similar to what the `__str__` method in `Item` returns. 

If you refactor `__str__` in the `Item` class so that you don't hardcode the word "item" in that string, then you can reuse the functionality here. 

```suggestion
        return f"{super().__str__()} It is made from {self.fabric} fabric."
```